solutions/cses-2079.mdx

```diff
@@ -126,5 +126,43 @@ public class Centroid {
 ```
 
 </JavaSection>
+	
+<PySection>
+
+```py
+	
+MAX_N = 200001
```

Comment:
why +1?

Reply:
We will be getting Runtime error if we dont do that.  We are filling the values in g and subtree_size in 1 index based way

Reply:
can you make the values 0-indexed then?

Reply:
In general such problems are solved using 1 index based for better understanding. that is why the commited code that are present in Java and C++ are done in the same way.

Reply:
debatable
in that case remove the + 1 and just use `MAX_N + 1` everywhere else instead